tests/cancun/eip4844_blobs/test_blobhash_opcode_contexts.py

```diff
@@ -100,7 +100,7 @@ def deploy_contract(
                 )
                 bytecode = (
                     sum(
-                        Op.MSTORE(offest=0, value=index)
```

Comment:
:grimacing: the offset defaults to 0, so no harm done.

Created a follow-up issue:
- #1716.